ep_gui/resources/lang/ep_gui.en-US.rc

```diff
@@ -362,16 +362,16 @@ BEGIN
     IDS_OTHER_PWRBTNACTION_2    "Shut down (default)"
     IDS_OTHER_PWRBTNACTION_4    "Restart"
     IDS_OTHER_NOREDIRECT        "Prevent the following Control Panel links from being redirected to the Settings app:"
-    IDS_OTHER_NOREDIRECT_SYSTEM "System / About page"
-    IDS_OTHER_NOREDIRECT_PROGRAMS "Programs and Features"
-    IDS_OTHER_NOREDIRECT_DATETIME "Adjust date/time"
-    IDS_OTHER_NOREDIRECT_TRAYICONS "Customize notification icons"
+    IDS_OTHER_NOREDIRECT_SYSTEM "System/About page"

```

Comment:
Should be shortend to "System" since windows 10 had this behavior aswell.

Reply:
Finally don't change "System / About page"